src/main/target/common_post.h

```diff
@@ -266,6 +266,10 @@
 #define USE_SERIAL_4WAY_BLHELI_INTERFACE
 #endif
 
+#if !defined(USE_PWM_OUTPUT) && defined(USE_SERIAL_4WAY_BLHELI_INTERFACE)
```

Comment:
No need to check `&& defined(USE_SERIAL_4WAY_BLHELI_INTERFACE)` here as `undef` is ignored if there is no define.

Reply:
yeah, I did wonder about that, wasn't sure if it made the code more readable or not.  I'll update it.

Reply:
done